### PR TITLE
enable streaming for ts list keys

### PR DESCRIPTION
**Pull request by hmmr**
RTS-697, RTS-718.

This PR uses the new `{o, ...}` sext-encoded keys for TS, and enables streaming list_keys.

Complementary PRs: https://github.com/basho/riak-erlang-client/pull/253 and https://github.com/basho/riak_test/pull/953.

### CODE REVIEW

File: src/riak_kv_eleveldb_backend.erl [hunk 1]
```diff
@@ -954,9 +955,9 @@ orig_to_object_key(Bucket, Key) ->
 %% be round-tripped (as that would then be a binary-wrapping a sext-encoded
 %% TS key - for an extra 9 bytes used).
 %%
-to_object_key({BucketType, BucketName}, {Family, Series, Timestamp}) ->
-    EncodedBucketType = sext:encode(BucketType),
-    EncodedBucketName = sext:encode(BucketName),
+to_object_key({TableName, TableName}, {Family, Series, Timestamp}) ->
+    EncodedBucketType = % sext:encode(BucketType),
```
Comment (jonmeredith) [inline src/riak_kv_eleveldb_backend.erl:958]: clever, I like it.

File: src/riak_kv_eleveldb_backend.erl [hunk 2]
```diff
@@ -49,6 +49,7 @@
 
 -export([data_size/1]).
 
+-compile([export_all]).
```
Comment (alexmoore) [inline src/riak_kv_eleveldb_backend.erl]: Do we need this?

Comment (hazen) [inline src/riak_kv_eleveldb_backend.erl]: Excellent point.  :-1:   Let's export what we really need.

**Comment by hazen** [comment]
:+1: 87d4a5d

File: src/riak_kv_eleveldb_backend.erl (hunk 2 above)
Comment (hmmr) [inline src/riak_kv_eleveldb_backend.erl]: There is no explanation I can give for it to be there. It just emerged from the conflict resolution pains. I have removed it.

**Comment by hazen** [comment]
@borshop merge

**Pull request merged**